web/Web/web/static/js/document-viewer.js

```diff
@@ -178,6 +195,14 @@ docView.prototype = {
               rel_val = 0;
               $(options.documentNonRelButtonSelector).each(function() {_linkJudgingButtons(this, rel_val)});
               break
+            case options.docViewNextDocButtonSelector:
```

Comment:
We don't need to add these in the `forEach` loop. 

I think this would work too. 
```
_linkNextDocButton($(options.docViewNextDocButtonSelector));
_linkPreviousDocButton($(options.docViewPreviousDocButtonSelector));
_linkSortByRelevanceButton($(options.sortReviewedDocumentsSelector));
```

Reply:
Done